pkg/parser/parser.y

```diff
@@ -15484,6 +15490,60 @@ UnlockStatsStmt:
 		}
 	}
 
+RefreshStatsStmt:
+	"REFRESH" "STATS" RefreshObjectList
+	{
+		$$ = &ast.RefreshStatsStmt{
+			RefreshObjects: $3.([]*ast.RefreshObject),
+		}
+	}
+
+RefreshObjectList:
+	RefreshObject
+	{
+		$$ = []*ast.RefreshObject{$1.(*ast.RefreshObject)}
+	}
+|	RefreshObjectList ',' RefreshObject
+	{
+		$$ = append($1.([]*ast.RefreshObject), $3.(*ast.RefreshObject))
+	}
+
+RefreshObject:
+	'*'
```

Comment:
This syntax is not included in the issue. And I think it's confusing to treat `*` as global level or database level. Can we remove it?

Reply:
Yes, I have discussed it with the optimizer team. We decided to remove it. However, MySQL uses it to refer to all tables in the current database. See: https://dev.mysql.com/doc/refman/5.7/en/grant.html